.github/workflows/probot-check-group.yml

```
@@ -0,0 +1,16 @@
+name: Probot
+
+on:
+  pull_request:
+
```

```suggestion

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
  cancel-in-progress: true
  
```
to limit only one per PR